.github/workflows/main.yml

```
@@ -157,48 +156,48 @@ jobs:
+          name: cypress-screenshots
+          path: src/test/cypress/cypress/screenshots
 
       - name: Load tests
         if: ${{ github.event.inputs.gatling  == 'true' || github.event_name == 'schedule' }}
         run: ./CICD/github/launch_load_tests.sh
 
-      - name : Publish Documentation
-        if : ${{ github.event.inputs.doc  == 'true' || github.event_name == 'schedule'}}
+      - name: Publish Documentation
+        if: ${{ github.event.inputs.doc  == 'true' || github.event_name == 'schedule'}}
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:8: Double quote to prevent globbing and word splitting

(shellcheck)